libraries/AP_Scripting/applets/UniversalAutoLand.lua

```diff
@@ -0,0 +1,139 @@
+--[[ Upon Arming , creates a four item mission consisting of: NAV_TAKEOFF, DO_LAND_START,Final Approach WP opposite bearing from HOME of heading used during takeoff, at TKOFF_ALT or SCR_USER3 above home, SCR_USER2 or 2X TKOFF_DIST, and a LAND waypoint at HOME and stops until next disarm/boot. SCR_USER1 is used to enable or disable it.
+--]]
+
+--local variablees
+
+SCR_USER_EN = Parameter("SCR_USER1") -- enable this script
```

Comment:
should create a param table for an applet I think, with param docs

Reply:
done